packages/jest-editor-support/src/__tests__/SnapshotSupport-test.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+'use strict';
+
+const path = require('path');
+const SnapshotSupport = require('../SnapshotSupport');
+
+const snapshotFixturePath = path.resolve(__dirname, 'fixtures/snapshots');
+
+test('snapshot-test.example', () => {
+  const filePath = path.join(snapshotFixturePath, 'snapshot-test.example');
+  const required = require(
```

Comment:
"required" seems a bit odd. Maybe "expected" (this is what it calls when the assertion fails) or "snapshotData"/"snapshotFile"/"snapshots"?